core/src/main/scala/org/apache/gearpump/cluster/ClusterMessage.scala

```diff
@@ -88,6 +89,7 @@ object MasterToAppMaster {
   case class AppMasterDataDetail(appId: Int, appDescription: Application)
   case class ReplayFromTimestampWindowTrailingEdge(appId: Int)
   case class GetAppDataResult(key: String, value: Any)
+  case class WorkerList(workers: List[Int])
   case object AppDataReceived
 }
 
```

Comment:
Right now we extend the core protocol (AppMaster -> Master) but it's only used within experimental. We should have an easier way to extend the protocol for custom appmasters.